src/engraving/libmscore/part.h

```diff
@@ -70,21 +70,24 @@ class Part final : public ScoreElement
     QString _partName;              ///< used in tracklist (mixer)
     InstrumentList _instruments;
     QList<Staff*> _staves;
-    QString _id;                    ///< used for MusicXml import
-    bool _show;                     ///< show part in partitur if true
-    bool _soloist;                  ///< used in score ordering
+    ID _id = INVALID_ID;             ///< used for MusicXml import
```

Comment:
We don't seem to have settled on a preference for using = vs { } for member initialization...I'd been using { } everywhere as that seemed to be the most common.

Reply:
I like classical initialization (without any brackets, just = operator) better. It's more explicit